server/controllers.js

```diff
@@ -0,0 +1,70 @@
+const request = require('request');
+
+const createJsonController = endpoint => async (req, res) => {
+
+  const lastChar = endpoint.slice(-1);
+  if (lastChar !== '/') {
+    endpoint = endpoint + '/';
+  }
+
+  const { id } = req.params;
+  const options = {
+    'method': 'GET',
+    'uri': `${endpoint}${id}`,
+  };
+  request(options, (error, res2, body) => {
+    if (error) {
+      console.log(error);
+      res.sendStatus(500);
+      return error;
+    }
+    res.json(JSON.parse(body));
+  });
+};
+
+// Benny
+const workspace = createJsonController('http://54.193.132.156:4000/workspace-api/workspace');
+
```

Comment:
For resusability/changes of the code using env vars with fall backs might be a worthwhile approach for urls